src/frame/worker/GroupCrypto.ts

```diff
@@ -50,23 +51,33 @@ export function addAdminsToGroup(
 }
 
 /**
- * Decrypt the provided group private key and use it to generate transform keys to each public user key provided.
+ * Decrypt the provided group private key and use it to generate Schnorr signature and transform keys to each public user key provided.
  * @param {TransformedEncryptedMessage} groupPrivateKey Encrypted private key for group
+ * @param {PublicKey<string>}           groupPublicKey  Group public key of the group being added to
+ * @param {string}                      groupID         Group Id of the group being added to
  * @param {UserOrGroupPublicKey[]}      userKeyList     List of user public keys to generate transform keys to
  * @param {PrivateKey<Uint8Array>}      adminPrivateKey Private key of the group add who is adding these new members.
  * @param {SigningKeyPair}              signingKeys     Current users signing keys used to sign transform key
  */
 export function addMembersToGroup(
-    groupPrivateKey: TransformedEncryptedMessage,
+    encryptedGroupPrivateKey: TransformedEncryptedMessage,
+    groupPublicKey: PublicKey<string>,
+    groupID: string,
     userKeyList: UserOrGroupPublicKey[],
     adminPrivateKey: PrivateKey<Uint8Array>,
     signingKeys: SigningKeyPair
 ) {
     return loadRecrypt()
         .flatMap((Recrypt) => {
-            return Recrypt.decryptPlaintext(groupPrivateKey, adminPrivateKey).flatMap(([_, key]) =>
-                Recrypt.generateTransformKeyToList(key, userKeyList, signingKeys)
-            );
+            return Recrypt.decryptPlaintext(encryptedGroupPrivateKey, adminPrivateKey).flatMap(([_, key]) => {
+                return Future.gather2(
+                    Recrypt.generateTransformKeyToList(key, userKeyList, signingKeys),
+                    Future.of(Recrypt.schnorrSignUtf8String(key, publicKeyToBytes(groupPublicKey), groupID))
```

Comment:
Similar to the other PR that Colt commented on, we should change this call here to do just move it down to the `map` below.

Reply:
done